Automattic/wp-calypso
# TimeSince: Refer to component directly in reader

### Summary
In #19812 I introduced `TimeSince`, a lift-and-shift from the original `reader/post-time` to a more generalised home in `/components` to allow for use in other areas of Calypso.

This PR looks to remove `reader/post-time` completely and refer to `TimeSince` directly in place of it's usage.

Is this something we'd prefer to do? Or would we rather keep the references as-is and keep `reader/post-time` as a sort of 'proxy' file as @samouri suggests here: https://github.com/Automattic/wp-calypso/pull/19812#discussion_r151202038 ?

### Testing
- Post Comments
  - Go to a post with comments ([calypso.live](http://calypso.localhost:3000/read/blogs/44010780/posts/31489#comments), [local](http://calypso.localhost:3000/read/blogs/44010780/posts/31489#comments))
  - Make sure those comments each show a human readable time/date

- Trackbacks
  - Visit a post that has trackbacks, here's an example: ([calypso.live](read/feeds/27030/posts/1555053762#comments), [local](http://calypso.localhost:3000/read/feeds/27030/posts/1555053762#comments))
  - Make sure those trackbacks show a human readable time/date

- Combined Card Posts
  - Find a 'Combined Card' in reader (bunched posts under one header).
  - Make sure the posts within the combined card show a human readable time/date

- Full Post Header
  - Go to a posts full post view ([calypso.live](http://calypso.localhost:3000/read/blogs/44010780/posts/31489), [local](http://calypso.localhost:3000/read/blogs/44010780/posts/31489))
  - Make sure that the header of the post shows a human readable time/date

- Post Byline
  - Look for the byline of a post
    - <img width="839" alt="screen shot 2017-11-16 at 13 29 33" src="https://user-images.githubusercontent.com/4335450/32893684-52ea19d2-cad2-11e7-844c-bc473707d848.png">
  - Make sure the byline shows a human readable time/date
